agent/fw_drupal8.c

```diff
@@ -140,14 +140,17 @@ static void nr_drupal8_add_method_callback_before_after_clean(
     nrspecialfn_t after_callback,
     nrspecialfn_t clean_callback) {
   zend_function* function = NULL;
+  char* methodLC = NULL;
 
   if (NULL == ce) {
     nrl_verbosedebug(NRL_FRAMEWORK, "Drupal 8: got NULL class entry in %s",
                      __func__);
     return;
   }
 
-  function = nr_php_find_class_method(ce, method);
+  methodLC = nr_string_to_lowercase(method);
```

Comment:
Would a `NULL` check for `method` be good here?  Maybe we could short circuit out - and it would shield some log messages below from trying to pass `NULL` to a "`%s`" argument.

Reply:
I can add a `NULL` check, but it seems redundant - `method` is equal to `"invokeAllWith"` because [this](https://github.com/newrelic/newrelic-php-agent/blob/5383c94898dfc00492c05a874a70cc1c5766c8ca/agent/fw_drupal8.c#L645-L647) is the only place `nr_drupal8_add_method_callback_before_after_clean` is used.